backend/src/main/java/c/team/session/statistics/model/SessionAnswers.java

```diff
@@ -0,0 +1,30 @@
+package c.team.session.statistics.model;
+
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+import java.util.Map;
+
+@Data
+@Builder
+@Document
+@NoArgsConstructor
+@AllArgsConstructor
+public class SessionAnswers {
+    @Id
+    private String id;
+    private String sessionId;
+    private String questionId;
+    private Map<Integer, Integer> answerCounts;
+
+    public SessionAnswers(String sessionId, String questionId, Map<Integer, Integer> answerCounts) {
```

Comment:
A tego konstruktor w sumie nie jest przypadkiem pokrywany przez @Builder? Chyba, że jak masz tą lambdę w SessionsAnswersService to tam w lambdzie nie da się użyć buildera, dobrze myślę (a przynajmniej nie tak, by to ładnie wyglądało :P)?

Reply:
Tak, jest po to, żeby ładniej wyglądało w lambdzie